tests/README.md

```diff
@@ -2,20 +2,20 @@
 To ensure the whole stack is working as expected and track bugs, a layer based test architecture is needed on OCP. This plan consists of four layers. The first and second layers mentioned below, would be a part of the future automation testing framework on each OCP (x,y,z) release.
 
 #### L1 First Layer: Host OS and Driver Interfaces 
-Layer 1 test cases should be executed before deploying [Intel Device Plugins Operator](device_plugins/README.md) and after deploying OOT drivers like [Intel Data Center GPU Driver on OpenShift](https://github.com/intel/intel-data-center-gpu-driver-for-openshift/blob/main/README.md). It includes test cases :
+Layer 1 test cases should be executed before deploying [Intel Device Plugins Operator](../device_plugins/README.md) and after deploying OOT drivers like [Intel Data Center GPU Driver on OpenShift](../kmmo/README.md#managing-intel-dgpu-driver-with-kmm-operator). It includes test cases :
```

Comment:
I think it is good for us to use the absolute path like (/kmmo/README.md) in case we change this file's location.